Fabric.Identity.API/scripts/CatalystDosIdentity.psm1

```diff
@@ -599,6 +599,10 @@ function New-ApiRegistration {
     }
 
     try{
+        if(!($body -is [string])) {
+            $body = $body | ConvertTo-Json
```

Comment:
is this passed by reference?  if so we are changing the value of on input parameter.  Should we be using a local variable?

Reply:
nevermind, powershell passes by value.